src/js/modules/content/green/js/components/storeCounter.js

```diff
@@ -3,10 +3,11 @@ import { pluck } from 'rxjs/operators';
 
 class StoreCounterController {
     constructor($scope, experimentalService) {
+        this.$scope = $scope;
```

Comment:
why is this done ?
$scope is used in arrow function below and should have access to the variable

Reply:
👍 refactor miss.